api/proto/teleport/legacy/types/types.proto

```diff
@@ -1804,6 +1804,11 @@ message ProvisionTokenSpecV2Kubernetes {
     // This can be fetched from /openid/v1/jwks on the Kubernetes API Server.
     string JWKS = 1 [(gogoproto.jsontag) = "jwks,omitempty"];
   }
+  message OIDCConfig {
+    // Issuer is the URI of the OIDC issuer. It must have an accessible and
+    // OIDC-compliant `/.well-known/oidc-configuration` endpoint.
+    string Issuer = 1 [(gogoproto.jsontag) = "issuer,omitempty"];
```

Comment:
As an aside - should we be enforcing that users provide a `https` based issuer unless they've set some kind of InsecureAllowHTTPIssuer flag as well?

Reply:
Hmmm, that's a good point and should be easy enough to add. I'll make the change.

Reply:
I've added an HTTPS check and an override flag.